elastic/kibana
# [Security Solution] Update indexNames for general threat match timeline

## Summary

This pr fixes:
https://github.com/elastic/kibana/issues/122010

Step to verify:

1. Deploy a cloud instance with GCP Los Angeles / 8.0.0-SNAPSHOT
2. Go to the cloud Kibana you just created, then go to timeline / templates / prebuilt templates
3. Open each prebuilt templates, and make sure the issue is there. (current index is incorrectly set to `.siem-signals-default`)
4. Point your Elasticsearch to the one you just created. Clone the branch `8.0`, cherry-pick the fix from this branch, and run `8.0` branch
5. Go to local kibana, timelines / templates / prebuilt templates, and click on each of the **prebuilt** templates. Make sure the data source has no `.siem-signals-default`
